crates/solver/src/solver.rs

```diff
@@ -224,22 +225,31 @@ pub enum SolverType {
 #[derive(Clone)]
 pub enum SolverAccountArg {
     PrivateKey(PrivateKey),
+    Kms(Arn),
     Address(H160),
 }
 
 impl Debug for SolverAccountArg {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
         match self {
             SolverAccountArg::PrivateKey(k) => write!(f, "PrivateKey({:?})", k.public_address()),
+            SolverAccountArg::Kms(key_id) => write!(f, "KMS({:?})", key_id),
```

Comment:
Whenever you use a macro that uses the `{}` formatting syntax you can write the printed value directly inside the curly braces. e.g. `{key_id:?}`.
Usually `clippy` complains when one doesn't do that.